projects/canopy/src/lib/header/notification-badge/notification-badge.component.scss

```diff
@@ -4,7 +4,6 @@
   border-radius: 50%;
   color: var(--notification-badge-color);
   display: flex;
-  flex-grow: 0;
```

Comment:
How come this was removed?

Reply:
It is redundant as it's the default value for a flex item, not sure why it was there in the first place?